### PR TITLE
#505: Remove styles import (closes #505)

### LINKED ISSUE #505: Remove styles import
## requirements
Importing `.scss` files should be delegated to the final user of the react-component, and not to the component itself. Moreover, component `.scss` file should define variables that can be overwritten when the style is imported. 

## specs
- Remove `import *.scss` from every component `.js` file
- Define component-specific variables in SASS files. `.scss` files should not import from `variables.scss` 

## misc
{optional: other useful info}

### PR DESCRIPTION
**Pull request by veej**
Issue #505

### tests performed
- `npm start`
- `examples look fine` ✅ 

> An example of partial test is a field displaying 4 possible values. If 3 values are tested, with screenshots, and 1 is not, then it should be mentioned here.}

### CODE REVIEW

File: src/Divider/divider.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-@import '../utils/theme/variables.scss';
+$color: #E6E6E6;  // pearl
```
Comment (giogonzo) [inline src/Divider/divider.scss]: shouldn't we set all of these variables with a `!default` flag to make them overridable?

e.g.
```scss
$color: #E6E6E6 !default;
```

File: src/Panel/tabbedPanel.scss [hunk 2]
```diff
@@ -1,4 +1,6 @@
-@import '../utils/theme/variables.scss';
+$tabbed-panel-tab-color: #3f4e49 !default;       // $gunmetal
```
Comment (FrancescoCioria) [inline src/Panel/tabbedPanel.scss]: we're already in the context of "tabbed-panel", you can simply rename them as "tab-color" and "tab-color-active"

File: src/DropdownMenu/menu.scss [hunk 3]
```diff
@@ -1,14 +1,19 @@
-@import '../utils/theme/variables.scss';
+$background: white !default;
+$box-shadow-color: #628292 !default;             // $lynch
+$menu-title-color: #bdc2ca !default;             // $silver
```
Comment (FrancescoCioria) [inline src/DropdownMenu/menu.scss]: also here, you could rename "menu-title-color" in "title-color".

(I'm not sure if "menu-item-..." should be renamed to "item-..." I'll leave the decision to you)

File: src/Panel/tabbedPanel.scss (hunk 2 above)
Comment (giogonzo) [inline src/Panel/tabbedPanel.scss]: is this true even in terms of `.scss` (or better, `sass-loader`) semantics? In other words, are we sure sass-loader is compiling each top-level `.scss` import alone or we must take care of namespacing ourselves?

Comment (giogonzo) [inline src/Panel/tabbedPanel.scss]: couldn't find an answer to this but I assume sass-loader is spawning a node-sass compilation for each top level import of a `.scss` from js, so we should be ok

**Comment by FrancescoCioria** [comment]
@veej your test plan is empty, did you run the examples to see that you're correctly importing every sass?

**Comment by FrancescoCioria** [comment]
nevermind, I tested them and it looks ok

**Pull request merged**